airflow/contrib/operators/gcs_to_gcs.py

```diff
@@ -62,6 +62,8 @@ class GoogleCloudStorageToGoogleCloudStorageOperator(BaseOperator):
         For this to work, the service account making the request must have
         domain-wide delegation enabled.
     :type delegate_to: str
+    :param last_modified_time: Timestamp in GMT to filter source object(s)
```

Comment:
in `GMT`??

Reply:
@kaxil Good catch, I need to use UTC instead of GMT but I should also clarify the behavior in the docs. By default if there is no tzinfo `is_updated_after()` [sets tzinfo to UTC.](https://github.com/apache/incubator-airflow/blob/master/airflow/contrib/hooks/gcs_hook.py#L269) I'll update to better describe this behavior.